runtime-transaction/src/transaction_meta.rs

```diff
@@ -22,7 +22,7 @@ pub trait StaticMeta {
     fn message_hash(&self) -> &Hash;
     fn is_simple_vote_transaction(&self) -> bool;
     fn signature_details(&self) -> &TransactionSignatureDetails;
-    fn compute_budget_instruction_details(&self) -> &ComputeBudgetInstructionDetails;
+    fn instruction_details(&self) -> &InstructionDetails;
```

Comment:
my one concern with the rename is that it seems like `signature_details` are also details about the instructions? `InstructionDetails` seems too generic almost, wdyt?

Reply:
If the details still refer to compute budget, I prefer the naming to mention it, lest we mistake it for a generic instruction.

Reply:
> my one concern with the rename is that it seems like signature_details are also details about the instructions? InstructionDetails seems too generic almost, wdyt?

This is the first step to combine signature details into instruction details (as the poc we discussed some times ago), hopefully saving one iteration over instructions when there is one "details" for runtime-transactions. 

> If the details still refer to compute budget, I prefer the naming to mention it, lest we mistake it for a generic instruction.

Good point, makes me think all "details" definition should be in `runtime-transaction` crate, could leave ops to "details" (such as get, try_from etc) in proper crates. 

I'll play around for both feedbacks.